ui/page/creatorMemberships/creatorArea/overviewTab/view.jsx

```diff
@@ -0,0 +1,150 @@
+// @flow
+import React from 'react';
+import { URL } from 'config';
+import { formatLbryUrlForWeb } from 'util/url';
+import * as ICONS from 'constants/icons';
+import * as PAGES from 'constants/pages';
+import Button from 'component/button';
+import CopyableText from 'component/copyableText';
+import ChannelThumbnail from 'component/channelThumbnail';
+import ButtonNavigateChannelId from 'component/buttonNavigateChannelId';
+import HelpHub from 'component/common/help-hub';
+import TruncatedText from 'component/common/truncated-text';
+import './style.scss';
+
+type Props = {
+  channelsToList: ?Array<ChannelClaim>,
+  onTabChange: (tabIndex: number) => void,
+  hasTiers: ?boolean,
+  // -- redux --
+  bankAccountConfirmed: ?boolean,
+  doSetActiveChannel: (claimId: ?string, override?: boolean) => void,
+};
+
+function OverviewTab(props: Props) {
+  const { channelsToList, onTabChange, hasTiers, bankAccountConfirmed, doSetActiveChannel } = props;
+  const [total, setTotal] = React.useState({ supporters: 0, income: 0, received: 0 });
+
+  function selectChannel(channelClaim) {
+    doSetActiveChannel(channelClaim.claim_id, true);
+    onTabChange(1);
+  }
+
+  React.useEffect(() => {
+    channelsToList &&
+      channelsToList.map((channel) => {
+        setTotal({ supporters: 0, income: 0, received: 0 });
+      });
```

Comment:
:small_blue_diamond:  Minor

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#when_not_to_use_map
<img width="374" alt="image" src="https://user-images.githubusercontent.com/64950861/191073375-9189a0dd-944e-4633-b00a-bc1227d9e186.png">

Reply:
Thanks, the entire loop is unnecessary anyways. It's just a placeholder. There's no dynamic data on this page yet :(